src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,11 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        StringBuilder reverseText = new StringBuilder();
+        String textBuf = text;
+        textBuf = textBuf.replaceAll("[^a-zA-Z0-9]","");
+        text = text.replaceAll("[^a-zA-Z0-9]","");
```

Comment:
@Valdes2 avoid using replaceAll twice

Reply:
why is that? bad practice?

Reply:
Ooo, i realized! Sorry, stupid question.